Makefile

```diff
@@ -428,6 +428,7 @@ clean:
 	@rm -f src/couch/priv/couchspawnkillable
 	@rm -f src/couch/priv/couch_js/config.h
 	@rm -f dev/*.beam dev/devnode.* dev/pbkdf2.pyc log/crash.log
+	@rm -f dev/erlserver.pem dev/couch_ssl_dist.conf
```

Comment:
Let's add those two files to .gitignore. We have ./dev/log/ and ./dev/lib/ already and we can add these alongside those.

Reply:
added